install/resources/mybb_theme.xml

```diff
@@ -9511,6 +9511,39 @@ if(use_xmlhttprequest == "1")
 	</tr>
 </table>
 </td>
+</tr>]]></template>
+<template name="post_recaptcha_v3" version="1823"><![CDATA[<tr id="captcha_trow">
```

Comment:
Invalid template name. Template name should start with `post_captcha_`. Thats a requirement.
So, the template name should be `post_captcha_recaptcha_v3`.

Reply:
Oh Wait!!! New template is not required, I believe ...